### PR TITLE
Localization and minor update

### PR DESCRIPTION
**Pull request by silentmark**
- change the way how effects are sent via socket to users - always send full effectData instead of sometimes effect data and sometimes uuids.
- check what type is radius - as parsing value as roll causes exception. 
- localizations

### CODE REVIEW

File: src/document/actor.js [hunk 1]
```diff
@@ -159,15 +159,23 @@ export class WarhammerActor extends WarhammerDocumentMixin(Actor)
             for (let uuid of effectUuids)
             {
                 let effect = await fromUuid(uuid);
+                let message = game.messages.get(messageId);
```
Comment (moo-man) [inline src/document/actor.js:162]: Why is this here? It's already declared above

File: src/document/actor.js [hunk 2]
```diff
@@ -159,15 +159,23 @@ export class WarhammerActor extends WarhammerDocumentMixin(Actor)
...
                 effectData.push(effect.convertToApplied(message?.system?.test));
             }
-
-            await CONFIG.ActiveEffect.documentClass.create(effectData, {parent: this, message : messageId});
-
+            for (let data of effectData)
+            {
+                await CONFIG.ActiveEffect.documentClass.create(data, {parent: this, message : messageId});
+            }
```
Comment (moo-man) [inline src/document/actor.js:168]: What is the reason for this?

File: src/document/actor.js (hunk 1 above)
Comment (silentmark) [reply src/document/actor.js]: because i'm an idiot ;-). removed.

File: src/document/actor.js (hunk 2 above)
Comment (silentmark) [reply src/document/actor.js:167]: for me - its for debugging purpose - it doesnt change that much in terms of performance, but its easier to verify which effect is corrupted.

Comment (moo-man) [reply src/document/actor.js:167]: Well, I can sympathize, but let's go back to one database call for this.

Comment (silentmark) [reply src/document/actor.js]: done
